### PR TITLE
Fix screeshare rap

### PR DESCRIPTION
**Pull request by GhaziTriki**
A fix for #7867

To test copy
`index.html.erb` to `/usr/local/bigbluebutton/core/playback/screenshare`
`publish/screenshare.rb` to `/usr/local/bigbluebutton/core/scripts/publish`
`process/screenshare.rb` to `/usr/local/bigbluebutton/core/scripts/process`
`screenshare.yml` to `/usr/local/bigbluebutton/core/scripts`

Then use `getRecordings` API to check the recorded meeting format.

### CODE REVIEW

**Comment by kepstin** [comment]
This PR includes a bunch of unrelated changes to other recording formats, and because you've re-indented the screenshare process script, the diff doesn't clearly show the changes you've made.

Can you please redo this PR without the formatting changes?

File: record-and-playback/screenshare/scripts/screenshare.yml [hunk 1]
```diff
@@ -15,41 +15,5 @@ layout:
       :y: 540
       :width: 240
       :height: 180
-
 formats:
-  - :mimetype: 'video/webm; codecs="vp9, opus"'
```
Comment (kepstin) [inline record-and-playback/screenshare/scripts/screenshare.yml]: It looks like you've removed the ability to change the generated video formats by editing the config file.
That ability should be kept - people should not have to edit the processing code to change the settings.

**Comment by GhaziTriki** [comment]
@kepstin The latest commit removes unrelated changes. The script works with the provided fixes. Ready for test and merge.

**Comment by ffdixon** [comment]
@kepstin Can you review

**Comment by kepstin** [comment]
So, there's still a remaining problem with this, because it looks like something in the BigBlueButton install scripts is editing the `screenshare.yml` file using the `yq` tool to add a "playback_host" value.

This changes the format of the file, removing quotes from the strings `:input` and `:output` in the `screenshare.yml` file. As a result the ruby yaml parser reads them as a symbol and causes this error during processing:

```
I, [2019-09-23T20:12:52.219468 #17716]  INFO -- : Performing post-processing step 0
I, [2019-09-23T20:12:52.220008 #17716]  INFO -- : Executing: mkclean --quiet input output
E, [2019-09-23T20:12:52.220613 #17716] ERROR -- : no implicit conversion of Symbol into String
E, [2019-09-23T20:12:52.220819 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback.rb:122:in `popen'
E, [2019-09-23T20:12:52.220866 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback.rb:122:in `exec_ret'
E, [2019-09-23T20:12:52.220896 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback/edl.rb:70:in `block (2 levels) in encode'
E, [2019-09-23T20:12:52.220956 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback/edl.rb:69:in `chdir'
E, [2019-09-23T20:12:52.220995 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback/edl.rb:69:in `block in encode'
E, [2019-09-23T20:12:52.221036 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback/edl.rb:56:in `each'
E, [2019-09-23T20:12:52.221126 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback/edl.rb:56:in `each_with_index'
E, [2019-09-23T20:12:52.221181 #17716] ERROR -- : /usr/local/bigbluebutton/core/lib/recordandplayback/edl.rb:56:in `encode'
E, [2019-09-23T20:12:52.221225 #17716] ERROR -- : process/screenshare.rb:146:in `block in <main>'
E, [2019-09-23T20:12:52.221272 #17716] ERROR -- : process/screenshare.rb:140:in `each'
E, [2019-09-23T20:12:52.221316 #17716] ERROR -- : process/screenshare.rb:140:in `each_with_index'
E, [2019-09-23T20:12:52.221366 #17716] ERROR -- : process/screenshare.rb:140:in `<main>'
```

@ffdixon I think this is caused by something in the bbb-conf script? It should only be setting `playback_host` in the file `bigbluebutton.yml`, but instead it seems to be modifying all the .yml files incorrectly.

**Comment by kepstin** [comment]
I've confirmed with @ffdixon that the packaging stuff causing the symbol issue has been fixed, so this is good to merge.

**Pull request merged**